### PR TITLE
fix(Video): check if onReady exists

### PR DESCRIPTION
**Pull request by natalieweesh**
## Overview
fix Video component - it is trying to call onReady(player) when there is no onReady prop which causes an error

### PR Checklist

- [ ] Related to Abstract designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### Description

---

## Merging your changes

When you are ready to merge to master and publish your changes, use the "squash and merge" button in GitHub, and follow the [PR Title Guide](https://github.com/Codecademy/client-modules#pr-title-guide) to format your PR title and write your PR merge description.

**IMPORTANT:** If your PR contains breaking changes, please remember to follow the instructions for breaking changes!

### CODE REVIEW

File: packages/gamut/src/Video/index.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const Video: React.FC<VideoProps> = ({
         muted={muted}
         playIcon={<OverlayPlayButton />}
         onReady={(player: ReactPlayer) => {
-          onReady(player);
+          onReady && onReady(player);
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/Video/index.tsx:56]: ⛳ ✨ 

```suggestion
          onReady?.(player);
```

Comment (natalieweesh) [reply packages/gamut/src/Video/index.tsx:56]: fancy! 🥂

**Comment by JoshuaKGoldberg** [comment]
😥  we never enabled strict mode in the root `tsconfig.json`, that's why TS never complained! Wowee. I'll do that separately...

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.31-alpha.346593.0<br/>@codecademy/gamut@9.2.4-alpha.346593.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef4b054b754bacae73472f9--codecademy-styleguide.netlify.app>https://5ef4b054b754bacae73472f9--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef4b054b754bacae73472f9>Deploy Logs</a></p>

**Pull request merged**